pos_refresh_customer/static/src/js/models.js

```diff
@@ -0,0 +1,54 @@
+// Copyright 2022 Coop IT Easy SCRLfs
+// License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+odoo.define("pos_customer_wallet.screens", function (require) {
+    "use strict";
+    var models = require("point_of_sale.models");
+    var rpc = require("web.rpc");
+
+    models.PosModel = models.PosModel.extend({
+        load_partners_by_ids: function (ids) {
+            var self = this;
+            var def = new $.Deferred();
+            var fields = _.find(this.models, function (model) {
+                return model.model === "res.partner";
+            }).fields;
+
+            rpc.query(
+                {
+                    model: "res.partner",
+                    method: "read",
+                    args: [ids, fields],
+                },
+                {
+                    timeout: 3000,
+                    shadow: true,
+                }
+            ).then(
+                function (partners) {
+                    // Context: `add_partners` fetches partner models from the
+                    // Odoo database and inserts those models into its own db.
+                    // As an efficiency measure, it does not insert partner
+                    // models if their `write_date` is LESS recent than the last
+                    // time it recorded that a partner model was successfully
+                    // added to the pos db. Computed fields do not normally
+                    // increase `write_date`. Therefore, to make sure that
+                    // computed fields also get updated, just reset the cached
+                    // date to the Unix epoch.
+                    self.db.partner_write_date = "1970-01-01 00:00:00";
```

Comment:
This works, but I'm not happy with this, and needs some tweaking.

It introduces one specific problem:

1. `add_partners()` is run on all partners. All is well and `partner_write_date` is consistent. Let's say it's equal to `2022-01-01 11:00:00`.
2. Partner 1 is edited at 13:00.
3. Partner 2 is edited at 14:00.
4. `load_partners_by_id()` (or `add_partners()`) is run on [Partner 2]. `partner_write_date` is now `2022-01-01 14:00`
5. `add_partners()` is run on [Partner 1]. This partner is skipped, because 13:00 is less recent than 14:00.

Part of this problem is a weird design decision by upstream `point_of_sale`. It kind of assumes that you're running `add_partners` on all partners (this is approximately what `load_new_partners()` does). We're not doing that.

Maybe we should remember `partner_write_date`'s value before editing it to the Unix epoch, and then `partner_write_date` it to its prior value after doing our thing?